lab_1_keywords_tfidf/main.py

```diff
@@ -82,7 +124,15 @@ def calculate_tf(frequencies: dict[str, int]) -> Optional[dict[str, float]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(frequencies, dict) and type_of_elements(frequencies, tuple, str, int):
+        term_freq_dict = {}
+        for element in frequencies.items():
+            term_freq = element[1] / len(frequencies)
```

Comment:
isn't len(frequencies)  a number of unique words in the text? In TF formula, we divide each word's frequency by the length of the whole text, not just number of unique terms in that text